extension/intents/navigation/navigation.js

```diff
@@ -34,32 +34,31 @@ function saveTabQueryToDatabase(query, tab, url) {
 intentRunner.registerIntent({
   name: "navigation.navigate",
   async run(context) {
-    const query = context.slots.query.toLowerCase();
-    const result = await browser.storage.sync.get("pageNames");
-    const pageNames = result.pageNames;
+    const name = context.slots.name.toLowerCase();
```

Comment:
I don't think these changes to navigation.js are necessary or related to the rest of the changes